apps/dashboard/app/views/dashboard/index.html.erb

```diff
@@ -244,6 +244,15 @@ $.get(perfUrl, function(jsonData){
     },
     title: function(){
       return "XDMoD - Center Report Card - " + startOfYear + " to " + today;
+    },
+    app_kernel_url: function(){
+      var link_info = {
```

Comment:
I don't feel AppKernel information is useful to users. It's more geared towards staff.

Reply:
This is true. Once the integration work with these graphs/tables are complete, we will have a quicker path forward to add other ones, add ones for staff vs user, etc. That is the discussion we should have next Monday with the rest of the team.